docs/en/operations/system-tables/blob_storage_log.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: /en/operations/system-tables/blob_storage_log
+---
+# Blob Storage Operations Log
+
+Contains logging entries with information about various blob storage operations such as uploads and deletes.
+
+Columns:
+
+- `event_date` ([Date](../../sql-reference/data-types/date.md)) — Date of the event.
+- `event_time` ([DateTime](../../sql-reference/data-types/datetime.md)) — Time of the event.
+- `event_time_microseconds` ([DateTime64](../../sql-reference/data-types/datetime64.md)) — Time of the event with microseconds precision.
+- `event_type` ([Enum8](../../sql-reference/data-types/enum.md)) — Type of the event. Possible values:
+    - `'Upload'`
+    - `'Delete'`
+    - `'MultiPartUploadCreate'`
+    - `'MultiPartUploadWrite'`
+    - `'MultiPartUploadComplete'`
+    - `'MultiPartUploadAbort'`
+- `query_id` ([String](../../sql-reference/data-types/string.md)) — Identifier of the query associated with the event, if any.
+- `thread_id` ([UInt64](../../sql-reference/data-types/int-uint.md#uint-ranges)) — Identifier of the thread performing the operation.
```

Comment:
could be also helpful to have `thread_name` column

Reply:
Added